libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -670,6 +694,21 @@ string ASTJsonConverter::visibility(Declaration::Visibility const& _visibility)
 	}
 }
 
+string ASTJsonConverter::location( VariableDeclaration::Location _location)
+{
+	switch (_location)
```

Comment:
sry, i am still a bit clumsy with the git workflow and my forcepush deleted a comment by alex regarding whether location should also include an option for "calldata"

i am not sure. the parser only differentiates between these three, so maybe the right place to introduce this would be another issue for the parser?